src/test/kotlin/apply/domain/applicationForm/ApplicationFormTest.kt

```diff
@@ -0,0 +1,57 @@
+package apply.domain.applicationForm
+
+import apply.domain.recruitmentitem.Answer
+import apply.domain.recruitmentitem.Answers
+import org.assertj.core.api.Assertions.assertThatThrownBy
+import org.junit.jupiter.api.Assertions.*
+import org.junit.jupiter.api.BeforeEach
+import org.junit.jupiter.api.Test
+
+internal class ApplicationFormTest {
+    private lateinit var applicationForm : ApplicationForm
+
+    @BeforeEach
+    internal fun setUp() {
+        applicationForm = ApplicationForm(
+                1L,
+                1L,
+                "http://example.com",
+                Answers(mutableListOf(
+                Answer("스타트업을 하고 싶습니다.", 1L),
+                Answer("책임감", 2L)
+        )))
```

Comment:
```suggestion
        applicationForm = ApplicationForm(
            applicantId = 1L,
            recruitmentId = 1L,
            referenceUrl = "http://example.com",
            answers = Answers(mutableListOf(
                Answer("스타트업을 하고 싶습니다.", 1L),
                Answer("책임감", 2L)
            ))
        )
```

이 부분에 대해서 위와 같이 어떤 내용의 argument인지 명시를 해준다면 가독성이 올라갑니다..! 
특히 같은 형의 (1L, 1L) 인자라면 더더욱 이해하기가 쉬워지겠죠!!

[제이슨의 코멘트](https://github.com/woowacourse/service-apply/pull/58#discussion_r487623004)를 참고했습니다.